.travis.yml

```diff
@@ -11,13 +11,9 @@ matrix:
   allow_failures:
   - php: nightly
 
-env:
-  - COMPOSER_OPTS=""
-  - COMPOSER_OPTS="--prefer-lowest"
```

Comment:
If we use the ```--prefer-lowest```, it will use the PHPUnit ```1.4``` to do the tests.
And the ```php-7.2``` will trigger the deprecated ```each``` function error when using the PHPUnit ```1.4```.
You can see more details about this [Travis build log](https://travis-ci.org/peter279k/sessions/jobs/364930358).

Reply:
That is an issue _introduced_ by this PR, if you can't find a way around it without removing these tests then we won't be able to accept the PR

Reply:
I found that issue because of the defined PHPUnit version.
It will be fixed in the latest Travis build :+1: .